sass/layout/_sidebar-left.scss

```diff
@@ -32,6 +32,10 @@
     height: 100%;
     z-index: 12;
 
+    body.announcement-bar--fixed & {
+        height: calc(100% - #{$announcement-bar-height});
```

Comment:
does this help with the `Refresh the app` banner?

Reply:
yes, it's all part of the `AnnouncementBar` component